src/pudl/glue/epacems_unitid_eia_plant_crosswalk.py

```diff
@@ -0,0 +1,92 @@
+"""Extract, clean, and normalize the EPA-EIA crosswalk.
+
+This module defines functions that read the raw EPA-EIA crosswalk file, clean
+up the column names, and separate it into three distinctive normalize tables
+for integration in the database.
+
+The crosswalk file was a joint effort on behalf on EPA and EIA and is published on the
+EPA's github account at www.github.com/USEPA".
+"""
+import logging
+
+import pandas as pd
+
+import pudl
+from pudl.metadata.fields import apply_pudl_dtypes
+from pudl.workspace.datastore import Datastore
+
+logger = logging.getLogger(__name__)
+
+
+def extract(ds: Datastore) -> pd.DataFrame:
```

Comment:
There's no explicit need for the extract/transform steps to be broken out into their own functions. Since each of these functions are only a couple of lines, and not really meant to be reusable for any other purpose it would be fine to just have all of the operations in a single function.

Reply:
Ok I thought about doing this, but wasn't sure. Nice to have confirmation here, I'll put it all in the transform.

Reply:
Seems like `grab_split_clean()` could also be rolled into the same single `transform()` function.

Reply:
I kind of like the idea of mimicking the extract and transform process used by the non-glue tables. I.e.: at least keep the extract and the transform function for the table if not a third function to roll them all together.